web/sites/default/config/core.base_field_override.node.alert.menu_link.yml

```diff
@@ -11,7 +11,7 @@ bundle: alert
 label: 'Menu link'
 description: 'Computed menu link for the node (only available during node saving).'
 required: false
-translatable: false
+translatable: true
```

Comment:
This one is a little odd. Alerts don't use the menu link field. (It's not even displayed as we never intend someone to go to the default display. 🤔 )

Reply:
Yeah, I think I was just following the pattern of other entities that do use that field so they have it checked. I guess the question is whether or not having it translatable matters in this case since it's not used.

Reply:
Just pushed an amended commit that makes all of the menu link fields non translatable.